hermes-common/src/test/groovy/pl/allegro/tech/hermes/domain/filtering/AvroPathMessageFilterSpec.groovy

```diff
@@ -20,6 +20,9 @@ class AvroPathMessageFilterSpec extends Specification {
                 "id": "0001",
                 "type": "donut",
                 "name": "Cake",
+                "flag": {
```

Comment:
Can we also make a test for empty map and for map containing more than 1 element?

Reply:
Ofc. Done